lib/winston/transports/file.js

```diff
@@ -458,12 +476,16 @@ module.exports = class File extends TransportStream {
    * @param {function} callback - Callback for when the current file has closed.
    * @private
    */
-  _endStream(callback) {
-    this._stream.unpipe(this._dest);
-    this._dest.end(() => {
-      this._cleanupStream(this._dest);
-      callback();
-    });
+  _endStream(callback = () => {}) {
```

Comment:
This isn't really related, I just noticed it was possible to make this throw if _dest or callback weren't set, so I added some bumpers...